guides/v3.5.0/tutorial/service.md

```diff
@@ -108,56 +61,59 @@ Let's get started creating our service by generating it through Ember CLI,
 which will create the service file, as well as a unit test for it.
 
 ```bash
-ember g service maps
+ember g service map-element
 ```
 
 Now implement the service as follows.
+The main API will be an [async function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) called `getMapElement`,
```

Comment:
Introducing async/await here.  We may want to explain it some more.  Also linking to an explanation.  I've actually created in issue for introducing it earlier, #228